manifests/init.pp

```diff
@@ -24,6 +24,8 @@
 #
 # $group::              The Katello system user group
 #
+# $deployment_url::     The URL katello is deployed under
```

Comment:
This is not exactly what this is intended for and I don't think we should expose this. This is something we have to configure for Candlepin and is specific to our use case.